ethereumj-core/src/main/java/org/ethereum/crypto/ECKey.java

```diff
@@ -345,9 +516,13 @@ public ECPoint getPubKeyPoint() {
      * @throws java.lang.IllegalStateException if the private key bytes are not available.
      */
     public BigInteger getPrivKey() {
-        if (priv == null)
+        if (privKey == null) {
+            throw new MissingPrivateKeyException();
+        } else if (privKey instanceof BCECPrivateKey) {
+            return ((BCECPrivateKey) privKey).getS();
```

Comment:
I see that `BCECPrivateKey.getS()` and `BCECPrivateKey.getD()` return the same value. Here both of them are used, is it intentionally? 
I'm also wondering if this is the same matter which just has different names in different interfaces or there is a subtlety?